source/operations/network-encryption.rst

```diff
@@ -106,6 +130,10 @@ Enabling TLS
    When the MinIO container starts, the server searches the specified location for certificates and uses them to enable TLS.
    Applications can use the ``public.crt`` as a trusted Certificate Authority to allow connections to the MinIO deployment without disabling certificate validation.
 
+   If you are reconfiguring an existing deployment that did not previously have TLS enabled, update ``MINIO_VOLUMES`` to specify ``https`` instead of ``http``.
```

Comment:
I guess Find and Replace these if we want to link to the envvar for `MINIO_VOLUMES`.